estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': "Real Estate",
+    'version': '1.0',
+    'depends': ['base'],
+    'author': "drat-odoo",
+    'category': 'Category',
```

Comment:
```suggestion
    'category': 'Estate',
```
The `category` should be relevant to the module, it should be indicationg where it relates with.
Here is a [reference](https://github.com/odoo/odoo/blob/94ed7173de4d6a73c5cfa6843b3628084248f18d/addons/account/__manifest__.py#L15)

Reply:
Okay sir